### PR TITLE
Explain how to run locally

### PR DESCRIPTION
**Pull request by matks**
Explain how to run locally

### CODE REVIEW

**Review by Progi1984 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -18,6 +21,27 @@ We write about everything that happens to the PrestaShop software:
 
 This devblog also hosts How-Tos on certain features, before they find their way in the Developer Guide or the Designer Guide.
 
+## Run locally
+
+build.prestashop.com is hosted by GitHub Pages and generated by [Jekyll](https://jekyllrb.com/) which is a Ruby static site generator.
```
Comment (PierreRambaud) [inline README.md]: it's generated by `github-pages` https://github.com/PrestaShop/prestashop.github.io/blob/master/Gemfile#L2
Yes it used Jekyll but the main package is https://github.com/github/pages-gem this is the reason why we can't use a better Ruby version or update Jekyll :/

File: README.md [hunk 2]
```diff
@@ -18,6 +21,27 @@ We write about everything that happens to the PrestaShop software:
...
+
+We recommend using Ruby 2.5 or 2.6.
```
Comment (PierreRambaud) [inline README.md]:
```suggestion
We recommend using Ruby 2.5.8
```
as it's defined here https://github.com/github/pages-gem/blob/master/.ruby-version

File: README.md [hunk 3]
```diff
@@ -18,6 +21,27 @@ We write about everything that happens to the PrestaShop software:
...
+
+### Setup
+
+```bash
+$ gem install --user-install bundler jekyll
```
Comment (PierreRambaud) [inline README.md]:
```suggestion
$ gem install bundler
```
is enough, the user-install is a user-specific configuration

File: README.md [hunk 4]
```diff
@@ -18,6 +21,27 @@ We write about everything that happens to the PrestaShop software:
...
+$ bundle install
```
Comment (PierreRambaud) [inline README.md:34]: it will install jekyll :wink:

**Review by PierreRambaud — CHANGES_REQUESTED** [review]
Nice idea, few comments :)

File: README.md (hunk 1 above)
Comment (matks) [reply README.md]:
```suggestion
build.prestashop.com is built and hosted by [GitHub Pages](https://github.com/github/pages-gem) which leverages [Jekyll](https://jekyllrb.com/) (a Ruby static site generator) abilities.
```

**Review by PierreRambaud — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @PierreRambaud & @matks

**Pull request merged**